src/idiomatic/foundations-api-design/predictable-api/naming-conventions/01-get.md

```diff
@@ -0,0 +1,31 @@
+---
+minutes: 2
+---
+
+# Get
+
+Getting an element from a collection or container.
+
+```rust
+// Safe, fallible indexing.
+let items: Vec<u8> = vec![1, 2, 3];
+let option = items.get(0); // &Vec<u8> -> Option<&u8>
+assert_eq!(Some(1), option);
+let failed_option = items.get(1000); // &Vec<u8> -> Option<&u8>
+assert_eq!(None, option);
+```
+
+<details>
+- Gets are trivial, they get a value!
```

Comment:
This isn't helpful ("gets are trivial") nor accurate ("they get a value" - actually they can return a reference).

Please ensure that you talk about the expected type signature.